### DISCUSSION TITLE
Any release date for 3.12.0?

### DISCUSSION THREAD

**Discussion by cristalp** [smallrye/smallrye-open-api#1939] [General]
Hi!

I'm waiting for a couple of reported bug fixes to be released  :-)  Can anyone tell me whether a release date is planned for 3.12.0?

Best regards,
Eric

**Comment by MikeEdgar** [comment]
@cristalp , [release 3.12.0](https://github.com/smallrye/smallrye-open-api/releases/tag/3.12.0) is now out :rocket:

**Comment by cristalp** [comment]
Great, thanks!
